findDuplicates.py

```diff
@@ -38,30 +41,36 @@ def output(text=None, level=0, end="\n", flush=False):
         print(text, end=end, flush=flush)
 
 
-def search_pattern(file_type):
-    return "*." + file_type
-
+def search_pattern(file_types):
+    pattern = '([^.].*)[.](' + file_types[0]
+    for file_type in file_types[1:]:
+        pattern += "|" + file_type
+    pattern += ")"
+    output("Search pattern: " + pattern, 2)
+    return re.compile(pattern, re.IGNORECASE)
 
-def make_common_name(file, file_type):
+def make_common_name(file):
     """
     Given a MusicFile, return the full path name minus the extension and any extra sequence characters
-    For example. /some/path/file.m4a, /some/path/file 1.m4a, and /some/path/file 2.m4a should all return
+    For example. /some/path/file.m4a, /some/path/file 1.m4a, and /some/path/file (2).m4a should all return
     /some/path/file
     """
-    return re.compile(f"( [\\d]|).{file_type}$").sub("", file.full_path_name)
+    return re.compile(f"( \d+| [(]\d+[)]|).[^.]+$").sub("", file.full_path_name)
```

Comment:
Can you please update to a raw string literal

```suggestion
    return re.compile(r"( \d+| [(]\d+[)]|).[^.]+$").sub("", file.full_path_name)
```